source/captiveportal/captive-portal-configuration.rst

```diff
@@ -0,0 +1,171 @@
+Configuring a Captive Portal Zone
+=================================
+
+When creating a captive portal zone, multiple settings can be edited,
+depending on your needs : 
```

Comment:
This sentence needs reworded to remove `your`. Actually `, depending on your needs` can be removed entirely.

See https://docs.netgate.com/pfsense/en/latest/references/style-guide.html#what-to-avoid